aesd-char-driver/main.c

```diff
@@ -124,7 +270,8 @@ void aesd_cleanup_module(void)
 	/**
 	 * TODO: cleanup AESD specific poritions here as necessary
 	 */
-
+	
+	aesd_circular_buffer_deinit(&aesd_device.buff1);
```

Comment:
@chaisuresh no need to check in the `.dwo` file below, you should add that to `.gitignore`